Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN apt-get update \
 		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
-ENV NODE_VERSION 4.0.0
+ENV NODE_VERSION 4.1.2
```

Comment:
Just tested `4.2.6` and works smoothly with our backend components. I'l open a new PR for 4.2.6 (to avoid confusion since I embedded 4.1.2 to the name of this one)